src/components/AboutMe/AboutMe.css

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
`- [Project page](https://www.wassup-42.students.nomoredomainssbs.ru/)
`

this link is not accessible 

* See here https://disk.yandex.ru/i/ezvn-CKWkKdchw

Reply:
Google cloud free tier expired just before you review :)